.github/workflows/PR-check-cmake.yml

```diff
@@ -108,6 +108,7 @@ jobs:
           New-Item -Path './build_dir' -ItemType Directory
           cmake -S . -B "./build_dir" `
             -G Ninja `
+            -DWITH_VCPKG_BREAKPAD=ON `
```

Comment:
```
   & 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\Launch-VsDevShell.ps1' `
            -SkipAutomaticLocation -Arch amd64 -HostArch amd64
```

on Windows ,this can be achieved by action      ` - uses: ilammy/msvc-dev-cmd@v1`

Reply:
on Windows
```
      - name: Upload packages
        env:
```
such action will use powershell,  

can configured to use bash shell as well

```
      - name: Upload packages
        shell: bash
        env:
```

Reply:
> ilammy/msvc-dev-cmd@v1

I don't like it. It calls the cmd equivalent of this command through layers of wrappers 😅 (And it probably doesn't work. I don't want to touch cmd.)

`Launch-VsDevShell.ps1` is written in Microsoft's doc, it is supported.

https://learn.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022#developer-powershell

Reply:
ok

Reply:
> can configured to use bash shell as well

Yes, I agree that that part of code uses too many powershell and looks unusual. It is probably a mistake 😅

I will change it to bash at another time maybe? It works for now.